horace_core/configuration/@hor_config/hor_config.m

```diff
@@ -122,6 +130,7 @@
     properties(Access=protected, Hidden=true)
         % private properties behind public interface
         mem_chunk_size_ = 1e7;
+        fb_scale_factor_ = 3;
 
```

Comment:
To maintain previous behaviour as much as possible, I would make it 10, as 10 chunks was comfortable value to make memory based cuts